lib/send_sms.rb

```diff
@@ -0,0 +1,30 @@
+# Download the twilio-ruby library from twilio.com/docs/libraries/ruby
+require 'dotenv/load'
+require 'twilio-ruby'
+
+class SMS
+
+  attr_reader :account_sid, :auth_token, :from, :to, :time
+
+  def initialize
+    @account_sid = ENV["ACCOUNT_SID"]
```

Comment:
Good use of environment variables :) but it'd be even better if those values were injected via the constructor, to avoid `SMS` internally relying on the environment, but leaving this decision to the caller upstream:

```ruby
SMS.new(
  ENV["ACCOUNT_SID"],
  ENV["AUTH_TOKEN"],
  ENV["TWILIO_MOBILE"],
  ENV["MY_MOBILE"]
)
```